torch/quantization/observer.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Lets go with observers that support symmetric quant for weights and asymmetric quant for activations. Also, lets go with just taking the min and max: clipping the weights can cause significant errors.  This is the default, we will add more observers in subsequent PRs.  A baseline observer can be the following:    
`def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_symmetric):
        super(Observer, self).__init__()
        self.dtype = dtype
        self.qscheme = qscheme
        assert (self.qscheme in (torch.per_tensor_affine,torch.per_tensor_symmetric)  \
            'Default Observer only works for per_tensor quantization schemes'
    

    def forward(self, x):
        self.min_val = torch.min(x).float()
        self.max_val = torch.max(x).float()

    def calculate_qparams(self):
        if self.dtype == torch.qint8:
            qmin, qmax = -128, 127
        else:
            qmin, qmax = 0, 255
      if self.q_scheme == torch.per_tensor_symmetric:
                max_val = torch.max(-self.min_val, self.max_val)
                min_val = -max_val*(128.0/127.0)
        n_levels = 255.0
        scale = (max_val - min_val) / n_levels
        zero_point = qmin - round(min_val / scale)
        zero_point = max(qmin, zero_point)
        zero_point = min(qmax, zero_point)
    
    return torch.tensor([scale, zero_point])

Reply:
symmetric quant is not supported in quantized kernels now?

Reply:
It is, symmetric quant is the default, asymmetric quant support is being added.